.gitignore

```diff
@@ -27,7 +27,7 @@ yarn-error.log*
 
 # local env files
 .env*.local
-
+.env
```

Comment:
revert this

Reply:
Well It need bcz .env file create should not be push

Reply:
Line 29 takes care of that

Reply:
Well it did  not work on my device !!!!!!!!!